lib/compiler/src/beam_ssa_pre_codegen.erl

```diff
@@ -2303,7 +2336,7 @@ reserve_yregs(#st{frames=Frames}=St0) ->
 
 reserve_yregs_1(L, #st{ssa=Blocks0,cnt=Count0,res=Res0}=St) ->
     Blk = map_get(L, Blocks0),
-    Yregs = beam_ssa:get_anno(yregs, Blk),
+    Yregs = ordsets:from_list(cerl_sets:to_list(beam_ssa:get_anno(yregs, Blk))),
```

Comment:
Is there a more efficient way to go from cerl_sets to ordsets? 🤔

Reply:
I don't think so.